README.md

```diff
@@ -5,35 +5,50 @@
 
 ## Example Opt Folder
+The CARMA Platform requires that some files be located in the /opt/carma directory so they can be found at runtime. The example_opt_carma folder in this repository contains an example of this folder's structure though some files cannot be included in this repo due to size or license restrictions. Therefore, the installation instructions should be consulted for proper setup. The folder is presented here as a supporting reference and used by development setup scripts.
 
-The CARMA Platform requires that some files be located in the /opt/carma directory so they can be found at runtime. The example_opt_carma folder in this repository contains an example of this folder's structure though some files cannot be included in this repo due to size or license restrictions. Therefore, the installation instructions should be consulted for proper setup. The folder is presented here as a supporting reference and used by development setup scripts. 
+## Current Status of Hybrid ROS 1/ROS 2 System
+The current CARMA Platform system operates as a hybrid of ROS 1 Noetic and ROS 2 Foxy components. While all core CARMA Platform components related to connected and automated vehicle capabilities (communications, localization, routing, trajectory generation, etc.) and all real-world physical sensor drivers are built for ROS 2 Foxy, a small amount of ROS 1 Noetic content still exists within the system. To enable bi-directional communication between ROS 1 Noetic and ROS 2 Foxy portions of the software system, the `ros1_bridge` package is used. The ROS 1 Noetic content that still exists within the CARMA Platform system is described below.
+
+With regards to the primary [carma-platform](https://github.com/usdot-fhwa-stol/carma-platform) repository, the following ROS 1 Noetic content still exists:
+- The `rosbag_mock_drivers` and `mock_lightbar_driver` packages, which serve as mock hardware drivers to support local integration testing. These are expected to be upgraded to ROS 2 Foxy in the near future.
+- The `carma_record` package, which enables ROS 1 rosbags to be recorded. This package is being preserved to enable legacy data analysis scripts to be used for system evaluation.
+- The ROS 1 rviz configuration in the `carma` package, which enables rviz to be run natively when CARMA Platform is deployed on a PC with a ROS 1 Noetic installed to its desktop. This is planned be removed after all ROS 2 Foxy packages are upgraded to ROS 2 Humble.
```

Comment:
I thought we already ported in rviz2. Is this statement still valid?

Reply:
We've created an rviz2 configuration to use, but we are still simultaneously maintaining support for the old ROS 1 rviz configuration until we update all vehicle PCs and VMs to have a desktop installation of ROS 2. I have updated the text to clarify that we are currently supporting both ROS 1 rviz and ROS 2 rviz2.